CHANGELOG.md

```diff
@@ -22,6 +22,18 @@ NativeScript CLI Changelog
 * [Fixed #4504](https://github.com/NativeScript/nativescript-cli/issues/4504): Custom tagged versions of android runtime are not supported
 * [Fixed #4510](https://github.com/NativeScript/nativescript-cli/pull/4510): Handle HTTP 304 response status code
 
+5.3.1 (2019, April 03)
+==
+
+### Implemented
+* [Implemented #4492](https://github.com/NativeScript/nativescript-cli/pull/4492): API(kinvey): provide correct data to preview-sdk based on the schema
+
+### Fixed
+* [Fixed #4370](https://github.com/NativeScript/nativescript-cli/issues/4370): NativeScript CLI installation fails on linux
+* [Fixed #4451](https://github.com/NativeScript/nativescript-cli/issues/4451): Error while trying to start application on Android emulator with API level Q
+* [Fixed #4483](https://github.com/NativeScript/nativescript-cli/pull/4483): Detection fixes for emulator/device
+* [Fixed #4494](https://github.com/NativeScript/nativescript-cli/pull/4494): Detect java correctly
```

Comment:
this was a fix for internal issue (not released on live), so you can remove it from changelog